src/configuration/services/pagespeed.md

```diff
@@ -0,0 +1,41 @@
+# PageSpeed Module
+
+PageSpeed is a family of 4 tools developped by Google Inc to optimize of websites performance.
+
+Pagespeed module, also known as mod_pagespeed, is an open source HTTP module for Apache or Nginx, which runs several optimizations on the resources of your website, on the server side.
+
+See the [PageSpeed docs](https://developers.google.com/speed/) for more information.
+
+## Supported versions
+
+* 1.12
+
+## Setup
+
+In your ``.platform/services.yaml``:
+
+```yaml
+pagespeed:
+    type: pagespeed:1.12-rc
```

Comment:
Hey @damz, any idea when this is going to become non-rc?

Reply:
If we're documenting it as RC, then this page should say clearly in the header section that it's experimental/unsupported/beta/whatever.